src/controllers/developers/developers.js

```diff
@@ -62,3 +58,19 @@ export const postDeveloper = async (req, res, next) => {
     return next(err);
   }
 };
+
+export const getDeveloper = async (req, res, next) => {
+  try {
+    const { developer } = res.locals;
+
+    return res.status(200).send({
+      message: 'Developer retrieved!',
```

Comment:
there's no need for this `message` field since we have a response status of 200 and the developer body, you can remove `message`